### PR TITLE
Update Japanese navigation bar and add external links to new Arabic podcast

### PR DESCRIPTION
**Pull request by jordit**
Resolves JIRA n/a

Overall changes
======
Shortened Gaza conflict link in the Japanese nav bar so that it fits in one single in a desktop view and added Spotify and Apple links to the Worth Paying Attention To Arabic podcast

Code changes
======

- edited the navigation section of src/app/lib/config/services/japanes.ts 
- added Spotify and Apple links for p0h6d6nm on src/app/routes/onDemandAudio/tempData/podcastExternalLinks/arabic.js
- added a podcast page for p0h6d6nm to data/arabic/podcasts for testing purposes

Testing
======
1. Open locally the Japanese front page the second item in the navigation () should read ガザ not イスラエル・ガザ戦争
2. Open /arabic/podcasts/p0h6d6nm locally, the page should include links to Spotifyand Apple 

image

image

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: data/arabic/podcasts/p0h6d6nm.json
Comment (karinathomasbbc) [inline data/arabic/podcasts/p0h6d6nm.json:1]: Is this file needed, or was it just added to check that the podcast page could render locally?

**Review by karinathomasbbc — APPROVED** [review]

File: data/arabic/podcasts/p0h6d6nm.json
Comment (jordit) [reply data/arabic/podcasts/p0h6d6nm.json:1]: hi @karinathomasbbc  I've added the podcast page so that I could test that the links I've added to the external podcast config page work as expected

Comment (karinathomasbbc) [reply data/arabic/podcasts/p0h6d6nm.json:1]: Thanks for explaining, happy to keep it as it might be useful in future 😸

**Review by louisearchibald — APPROVED** [review]

**Comment by jordit** [comment]
thanks @karinathomasbbc and @louisearchibald
